tests/Doctrine/Tests/DBAL/Platforms/DB2PlatformTest.php

```diff
@@ -377,6 +377,33 @@ public function testModifiesLimitQuery()
         );
     }
 
+    public function testModifiesLimitQueryWithOrderBy()
+    {
+        self::assertEquals(
+            'SELECT * FROM user ORDER BY id',
+            $this->_platform->modifyLimitQuery('SELECT * FROM user ORDER BY id', null, null)
+        );
```

Comment:
This test could use `@dataProvider` to make assertions independent on each other.